src/app/compare/compare.component.css

```diff
@@ -1,4 +1,19 @@
 #container {
   display: flex;
   background-color: white;
+  height: 94vh;
+}
+
+.trees {
+  min-height: 18vh;
```

Comment:
same

Over het algemeen is het niet fijn om dit soort hoogtes te gebruiken omdat er altijd nog elementen boven en onder komen, dan moeten ze meteen weer aangepast worden. 
Probeer de parent containers te stylen met flexbox of grid en gebruik hier gewoon 100% height.